mule-maven-plugin/src/main/java/org/mule/tools/maven/mojo/VerifyMojo.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Mule ESB Maven Tools
+ * <p>
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * <p>
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.tools.maven.mojo;
+
+import org.mule.tools.api.exception.ValidationException;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+
+@Mojo(name = "verify",
+    defaultPhase = LifecyclePhase.VERIFY,
+    requiresDependencyResolution = ResolutionScope.RUNTIME)
+public class VerifyMojo extends AbstractMuleMojo {
+
+  @Override
+  public void doExecute() throws MojoExecutionException {
+    getLog().debug("Verifying packaged files ...");
+    try {
+      if (!skipValidation) {
+        getProjectVerifier().verify();
+      } else {
+        getLog().debug("skipping verify validation for Mule application");
```

Comment:
Skipping?

Reply:
capitalized Skipping here and from where I copied the message.